service/view_building_coordinator.hh

```diff
@@ -14,19 +14,71 @@
 
 #include "dht/i_partitioner_fwd.hh"
 #include "locator/host_id.hh"
+#include "locator/tablets.hh"
+#include "mutation/canonical_mutation.hh"
 #include "schema/schema_fwd.hh"
+#include "service/migration_manager.hh"
+#include "service/raft/raft_group0.hh"
+#include "service/raft/raft_group0_client.hh"
+#include "service/topology_state_machine.hh"
 
 namespace service {
 
 struct view_building_target {
     locator::host_id host;
     unsigned shard;
 
+    view_building_target(locator::host_id host, unsigned shard) : host(host), shard(shard) {}
+    view_building_target(const locator::tablet_replica& replica) : view_building_target(replica.host, replica.shard) {}
+
     std::strong_ordering operator<=>(const view_building_target&) const = default;
 };
 
 using view_building_tasks = std::map<view_building_target, dht::token_range_vector>;
 using base_building_tasks = std::map<table_id, view_building_tasks>;
 using view_building_coordinator_tasks = std::map<table_id, base_building_tasks>;
 
+class view_building_coordinator : public migration_listener::only_view_notifications {
+    struct vbc_state {
```

Comment:
This struct is in `view_building_coordinator` namespace. Does it need `vbc_` prefix?